src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/SubmissionComponentSpec.scala

```diff
@@ -119,41 +119,35 @@ class SubmissionComponentSpec extends TestDriverComponentWithFlatSpecAndMatchers
     }
   }
 
-  "WorkflowComponent" should "let you modify Workflows within a submission" in withDefaultTestDatabase {
-    val workspaceContext = SlickWorkspaceContext(testData.workspace)
+  "WorkflowComponent" should "update the status of a workflow and increment record version" in withDefaultTestDatabase {
+    val workflowRecBefore = runAndWait(workflowQuery.listWorkflowRecsForSubmission(UUID.fromString(testData.submission1.submissionId))).head
 
-    val workflow0 = testData.submission1.workflows(0)
-    assertResult(Some(workflow0)) {
-      runAndWait(workflowQuery.get(workspaceContext, testData.submission1.submissionId, workflow0.workflowEntity.get.entityType, workflow0.workflowEntity.get.entityName))
-    }
+    runAndWait(workflowQuery.updateStatus(workflowRecBefore, WorkflowStatuses.Failed))
 
-    val workflow1 = testData.submission1.workflows(1)
-    assertResult(Some(workflow1)) {
-      runAndWait(workflowQuery.get(workspaceContext, testData.submission1.submissionId, workflow1.workflowEntity.get.entityType, workflow1.workflowEntity.get.entityName))
-    }
+    val workflowRecAfter = runAndWait(workflowQuery.listWorkflowRecsForSubmission(UUID.fromString(testData.submission1.submissionId))).filter(_.id == workflowRecBefore.id).head
 
-    val workflow2 = testData.submission1.workflows(2)
-    assertResult(Some(workflow2)) {
-      runAndWait(workflowQuery.get(workspaceContext, testData.submission1.submissionId, workflow2.workflowEntity.get.entityType, workflow2.workflowEntity.get.entityName))
-    }
+    assert(workflowRecAfter.status == WorkflowStatuses.Failed.toString)
```

Comment:
Add this above with the initial status (Submitted?) to show that it has changed by this point